htdocs/js/AchievementList/achievementlist.js

```diff
@@ -65,17 +65,12 @@
 			const filter_text = document.getElementById('filter_text');
 			const filter_category = document.getElementById('filter_category');
 			if (filter === 'selected' && !is_achievement_selected()) {
-				e.preventDefault();
 				e.stopPropagation();
```

Comment:
These are the lines, unsure why they didn't show up in my last review.

Reply:
Oh I added them (er deleted them) after your comment and then force pushed.